os/binfmt/libelf/libelf_cache.c

```diff
@@ -454,18 +436,17 @@ int elf_cache_read(int filfd, uint16_t binary_header_size, FAR uint8_t *buffer,
  *   OK (0) on Success
  *   Negative value on Failure
  ****************************************************************************/
-int elf_cache_init(int filfd, uint16_t offset, off_t filelen, uint8_t compression_type)
+int elf_cache_init(int filfd, uint16_t offset, off_t filelen)
 {
 	int ret = OK;
 
-	binfo("filfd: %d offset: %d filelen: %d compression_type: %d\n", filfd, offset, filelen, compression_type);
+	binfo("filfd: %d offset: %d filelen: %d\n", filfd, offset, filelen);
```

Comment:
Please change `offset: %d` to `offset: %u`, because `offset` is unsigned type.

Reply:
@kishore-sn Could you check Jeongchan's comment?

Reply:
Done